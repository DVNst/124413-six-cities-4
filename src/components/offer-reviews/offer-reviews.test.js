import React from "react";
import renderer from "react-test-renderer";
import OfferReviews from "./offer-reviews.jsx";

const reviews = [
  {
    id: 1,
    offersId: 1,
    userName: `User1`,
    rating: 5,
    text: `description user 1`,
    dateTime: `April 10, 2020`,
  }, {
    id: 2,
    offersId: 1,
    userName: `User2`,
    rating: 4,
    text: `description user 2`,
    dateTime: `March 20, 2020`,
  },
];

it(`Render OfferReviews`, () => {
  const tree = renderer.create(
      <OfferReviews
        reviews={reviews}
      />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
