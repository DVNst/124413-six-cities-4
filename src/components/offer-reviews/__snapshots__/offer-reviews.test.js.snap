// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render OfferReviews 1`] = `
<ul
  className="reviews__list"
>
  <li
    className="reviews__item"
  >
    <div
      className="reviews__user user"
    >
      <div
        className="reviews__avatar-wrapper user__avatar-wrapper"
      >
        <img
          alt="Reviews avatar"
          className="reviews__avatar user__avatar"
          height={54}
          src="img/avatar-max.jpg"
          width={54}
        />
      </div>
      <span
        className="reviews__user-name"
      >
        User1
      </span>
    </div>
    <div
      className="reviews__info"
    >
      <div
        className="reviews__rating rating"
      >
        <div
          className="reviews__stars rating__stars"
        >
          <span
            style={
              Object {
                "width": "100%",
              }
            }
          />
          <span
            className="visually-hidden"
          >
            Rating
          </span>
        </div>
      </div>
      <p
        className="reviews__text"
      >
        description user 1
      </p>
      <time
        className="reviews__time"
        dateTime="10-04-2020"
      >
        April 2020
      </time>
    </div>
  </li>
  <li
    className="reviews__item"
  >
    <div
      className="reviews__user user"
    >
      <div
        className="reviews__avatar-wrapper user__avatar-wrapper"
      >
        <img
          alt="Reviews avatar"
          className="reviews__avatar user__avatar"
          height={54}
          src="img/avatar-max.jpg"
          width={54}
        />
      </div>
      <span
        className="reviews__user-name"
      >
        User2
      </span>
    </div>
    <div
      className="reviews__info"
    >
      <div
        className="reviews__rating rating"
      >
        <div
          className="reviews__stars rating__stars"
        >
          <span
            style={
              Object {
                "width": "80%",
              }
            }
          />
          <span
            className="visually-hidden"
          >
            Rating
          </span>
        </div>
      </div>
      <p
        className="reviews__text"
      >
        description user 2
      </p>
      <time
        className="reviews__time"
        dateTime="20-03-2020"
      >
        March 2020
      </time>
    </div>
  </li>
</ul>
`;
